### ISSUE TITLE
CURRENT_TIMESTAMP issue

### ISSUE THREAD

**Issue by tonykor** [bizley/yii2-migration#73]
the timestamp row with default value CURRENT_TIMESTAMP generated incorrect, migration script also not identify for example timestamp(3) value (timestamp with microseconds is available since mysql 5.6.4 afaik)

`'timestamp' => $this->timestamp()->notNull()->defaultValue('CURRENT_TIMESTAMP(3)'), `

```
SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'timestamp'
The SQL being executed was: CREATE TABLE `test` (
	`timestamp` timestamp(0) NOT NULL DEFAULT 'CURRENT_TIMESTAMP(3)'
) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE=InnoDB'
```
must be

`'timestamp' => $this->timestamp(3)->notNull()->defaultExpression('CURRENT_TIMESTAMP(3)'), `

my command line arguments are:

`php yii migration/create-all --generalSchema=0`

Please get a look

**Comment by bizley** [comment]
Thanks for reporting, let me take a look and see what is going on.

**Comment by bizley** [comment]
I've fixed the precision error (release tomorrow) but the bug with simple value instead of expression might take more time since this is how [Yii 2 returns this information](https://github.com/yiisoft/yii2/blob/2.0.30/framework/db/mysql/Schema.php#L296-L298). I'll try to prepare PR for the framework fixing it but we will have to stick with some workaround for now.

**Comment by bizley** [comment]
https://github.com/yiisoft/yii2/issues/17744

**Comment by bizley** [comment]
Fixed in [3.6.4](https://github.com/bizley/yii2-migration/releases/tag/3.6.4) / [2.9.4](https://github.com/bizley/yii2-migration/releases/tag/2.9.4)
